tests/conftest.py

```diff
@@ -1596,8 +1610,26 @@ def verify_new_core_dumps(duthost):
     new_core_dumps = list(new_core_dumps)
 
     if new_core_dumps:
-        pytest.fail("Core dumps found. Expected: %s Found: %s. Test failed. New core dumps: %s" % (len(pre_existing_cores),\
-            len(cur_cores), new_core_dumps))
+        reload = True
+        logger.info("Core dumps found. Expected: %s Found: %s. New core dumps: %s" % (len(pre_existing_cores), len(cur_cores), new_core_dumps))
+
+    # Check if the running config is different from the golden config
+    if duthost.stat(path=CONFIG_DB_BAK).get('stat', {}).get('exists', False):
+        ansible_cmd = "ansible -m compare_config -i ../ansible/{} {}".format(inv_name, duthost.hostname)
```

Comment:
You can just use `duthosts.compare_config()` to call this ansible module. Please use `duthosts` instead of `duthost` to ensure that this check is performed on all DUTs on multiple-DUTs testbed.

The core dumps check also needs to be performed on all DUTs.

Reply:
done